pytorch_blade/benchmark/TorchBench/torchbench.py

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/env python
+import gc
```

Comment:
Is this file modified?

Reply:
Not changed, but not include in torchdynamo wheel.

Reply:
So we can remove this file as discussed offline?